apps/etterlatte-behandling/src/main/resources/db/migration/V153__populer_fom_paa_eksisterende_unntak.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE aktivitetsplikt_unntak
+SET fom = subquery.opprettet_date,
+    beskrivelse = beskrivelse || ' - FOM ble automatisk satt av Gjenny -'
```

Comment:
Tar imot forslag til tekst som legges til her.

Reply:
`Fra-dato` i stedet for `FOM`?